PropertyChanged.Fody/InvokerTypes.cs

```diff
@@ -2,7 +2,8 @@
 {
     String,
     BeforeAfter,
-    BeforeAfterGeneric,
+    BeforeAfterGenericMethod,
+    BeforeAfterGenericParameters,
```

Comment:
This one is *not* useful for the event invoker. It makes sense for the BeforeAfter handler, but not much for an event invoker.
It will fail if you have *any* property that is of a type other than `T` in your class. I'd remove this entirely.